maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/ValidationServiceTest.java

```diff
@@ -147,32 +142,32 @@ void givenInputWithExistingReservation_whenIsUserActionValidIsInvoked_thenExcept
         assertThatThrownBy(() -> validationService.isUserActionValid(TestModelDataBuilder.getUserValidationDTOWithReservation()))
                 .isInstanceOf(CrimeValidationException.class)
                 .extracting("exceptionMessage", InstanceOfAssertFactories.ITERABLE)
-                .contains(EXISTING_RESERVATION_SO_RESERVATION_NOT_ALLOWED);
+                .contains(ValidationService.USER_HAVE_AN_EXISTING_RESERVATION_RESERVATION_NOT_ALLOWED);
```

Comment:
Could you please use static imports for all the constants?

Reply:
Done